qt/python/mantidqt/widgets/plotconfigdialog/axestabwidget/presenter.py

```diff
@@ -293,3 +298,12 @@ def show_minor_ticks_checked(self, checked):
 
         if not checked:
             self.view.set_show_minor_gridlines(False)
+
+    def _apply_all_properties_with_errors(self):
+        """Call apply_all_properties, notifying the parent presenter of whether there were any errors"""
+        try:
+            self.apply_all_properties()
+        except Exception as exception:
```

Comment:
This is too broad, it will catch `OutOfMemory`, `ImportError`, `RecursionError` ...etc. that shouldn't be gracefully handled but instead throw, especially if Python is in an undefined state now

Could we reduce the scope down to the expected exceptions?